### DISCUSSION TITLE
0.13 Beta DB Location

### DISCUSSION THREAD

**Discussion by leadZERO** [blakeblackshear/frigate#8182] [Ideas]
I just wanted to voice an opinion on the new 0.13 change to move the database into the config folder. Personally this adds an issue that now I have to add some files in the config folder to my .gitignore where I track my configuration changes. This also kind of violates the best practice of not mixing configuration with data. 

A few thoughts I had...
1) Move the database to another root level /data folder
2) Move the database to /media/data, which could be mapped separately in docker-compose, but not sure how this would affect things like HassOS.
3) Add the option to use a database server in(stead/addition?) to the sqlite database.

Anyway, love all the work and support the devs bring in this project! Just an opinion :)

R

**Comment by NickM-27** [comment]
Frigate has always supported the ability to set a custom db location if you want to

The default location must be in a location that fits within the home assistant addon which means it must be in /config or /media

Given the issues noted in the release notes, it has been moved from /media to /config

This is explained in more detail at https://github.com/blakeblackshear/frigate/discussions/7799#discussioncomment-7038175
